src/core/cache/memoizer.ts

```diff
@@ -0,0 +1,21 @@
+import { memoizeOne } from 'core/memoizer';
+import { CacheKeyFragment, getCache } from '.';
+
+export default <TKey extends CacheKeyFragment[]>() => {
```

Comment:
Using two main types of caches in the table so far:

- Memoizer caches take in a key to get access to a memoized function that takes a different set of parameters. The key may remain the same but the params change. (e.g. evaluating the dropdown for each cell)
- Value caches take in a key and executes a function with the key parameters if the value is no present in the cache. The key and the params are always equal. (e.g. evaluating/binding the event handlers for each cell)